CAP/gap/UniversalObjects.gd

```diff
@@ -1098,6 +1098,9 @@ DeclareOperation( "AddInitialObjectFunctorial",
 DeclareOperationWithCache( "DirectSumOp",
                            [ IsList, IsCapCategoryObject ] );
 
+DeclareOperation( "DirectSumOp",
```

Comment:
How is this function supposed to be called? Do you call it explicitly with a category? Then one might want to name it DirectSum.